scripts/publish-plugin.sh

```diff
@@ -17,7 +17,7 @@ if [ "$(go env GOOS)" = "windows" ]; then
 fi
 
 (cd "${ROOT}/provider" && go build \
-   -ldflags "-X github.com/pulumi/pulumi-gcp/provider/pkg/version.Version=${VERSION}" \
+   -ldflags "-X github.com/pulumi/pulumi-gcp/provider/v2/pkg/version.Version=${VERSION}" \
    -o "${WORK_PATH}/pulumi-resource-gcp${BIN_SUFFIX}" \
    "${ROOT}/cmd/pulumi-resource-gcp")
```

Comment:
Have you tested this code path? I wouldn't be surprised if that last line needs to be updated. It looks like that's a package?

Reply:
I was able to run this exact script from my local machine and it works